src/mongocxx/client.hpp

```diff
@@ -107,6 +108,27 @@ class MONGOCXX_API client {
     MONGOCXX_DEPRECATED void* implementation() const;
 
     ///
+    /// Sets the read concern for this client.
+    ///
+    /// Modifications at this level do not effect existing databases instances that have have been
+    /// created by this client but do effect new ones as databases inherit the @c read_concern
```

Comment:
Gah!!! It was copy paste, I swear! This is one of my biggest pet peeves. Good catch :)